### PR TITLE
Updates to show we now host Ubuntu repos in Assured and changed URLs …

### PR DESCRIPTION
**Pull request by cantlep**
…to point to the CDS'

### CODE REVIEW

**Review by mikepukcloud — APPROVED** [review]
LGTM

File: articles/shared/shared-how-access-ubuntu-update-servers.md [hunk 1]
```diff
@@ -23,37 +23,36 @@ When you create a virtual machine (VM) with an Ubuntu licence on it, you need to
...
-## Assured OFFICIAL platform
+### Assured OFFICIAL platform
 
 UKCloud's Assured OFFICIAL platform is internet facing, so you can either configure your VM to connect to the internet, and use standard update tools such as apt-get to talk directly to the publicly available Ubuntu repositories or you can use the UKCloud-managed repository servers.
 
-## Elevated OFFICIAL platform
+### Elevated OFFICIAL platform
 
-UKCloud's Elevated OFFICIAL platform doesn't natively connect to the internet, and the PSN Protected network doesn't have any Ubuntu repository servers. To connect to an Ubuntu repository server for updates, you can use UKCloud-managed repository servers or a UKCloud Walled Garden.
+UKCloud’s Elevated OFFICIAL platform doesn't have native access to Ubuntu repository servers. To connect to an Ubuntu repository server for updates, you can use UKCloud-managed repository servers or a UKCloud Walled Garden.
```
Comment (cantlep) [inline articles/shared/shared-how-access-ubuntu-update-servers.md:32]: Apostrophe change for UKCloud's but not doesn't, etc...?

Comment (suehighmoor) [reply articles/shared/shared-how-access-ubuntu-update-servers.md]: fixed

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**